menhir-recover/attributes.ml

```diff
@@ -82,14 +91,22 @@ module type ATTRIBUTES = sig
       actions.
 
       expr:
-        LPAREN expr error { ... } [@recover.cost infinity]
+        LPAREN expr error { ... } [@recover.cost inf]
       ;
 
       It would not make much sense for the recovery to select an error rule.
       Associating an infinite cost to the production ensures that this never
       happen.
+
+      Also, you can tweak the default cost by the `recover.default_cost_of_production`
+      grammar attribute otherwise default cost is 1.
   *)
 
+  (** Default cost of reducing a production that is used if it has no
```

Comment:
Probably `Default cost of production reduce ...` is better

Reply:
fix thx